### DISCUSSION TITLE
v1.26.0_36-dev

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#539] [Announcements]
### Server
* Feature - Support tiff uploading by @panoti in https://github.com/immich-app/immich/pull/513
* Refactor - move constants into the shared library by @panoti in https://github.com/immich-app/immich/pull/522
* Fixed - Large file size cannot be saved to EXIF table due to incorrect data type (`integer` vs `bigint`) in the Exif table by @alextran1502 in https://github.com/immich-app/immich/pull/534

## Mobile
* Added dutch translation for Immich by @nicodier in https://github.com/immich-app/immich/pull/519
* Fixed - Mobile timeline crash when date group cannot be parsed by @alextran1502 in https://github.com/immich-app/immich/pull/530

## Web
* Feature - Video player is buffered for faster startup time and reduces network usage by @panoti in https://github.com/immich-app/immich/pull/520
* Fixed - Skipping assets when navigating with the keyboard by @alextran1502 in https://github.com/immich-app/immich/pull/531
* Migrate SvelteKit to the latest version `431` by @alextran1502 in https://github.com/immich-app/immich/pull/526
* Fixed - Show the first two letter of user first and last name when profile image not existed by @alextran1502 in https://github.com/immich-app/immich/pull/532
* Feature - Implemented notification box for web by @alextran1502 in https://github.com/immich-app/immich/pull/533
* Fixed - Limit number of file upload on web by @alextran1502 in https://github.com/immich-app/immich/pull/535
* Optimization - Added error handling notification by @alextran1502 in https://github.com/immich-app/immich/pull/536

## Support

If you find the project helpful and help you in some ways, you can support the project [one time](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502) or [monthly](https://github.com/sponsors/alextran1502) from Github Sponsor 

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

## F-Droid
You can get the app on F-droid by clicking the image below.

[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
    alt="Get it on F-Droid"
    height="80">](https://f-droid.org/packages/app.alextran.immich)

#### You can also download the app from Google Play Store [here](https://play.google.com/store/apps/details?id=app.alextran.immich) 

*The App version might be lagging behind the latest release due to the review process.*

<p align="left">
  <img src="https://user-images.githubusercontent.com/27055614/164953845-4749bccf-8f9d-45e1-ac41-61fe98d6e5b4.png" width="200" title="Google Play Store">
<p/>

#### You can download the app from Apple AppStore [here](https://apps.apple.com/us/app/immich/id1613945652):

<p align="left">
  <img src="https://user-images.githubusercontent.com/27055614/164953841-de6e3f0f-4fd4-4e25-8327-6c3ccc622fc2.png" width="200" title="Apple App Store">
<p/>

*The App version might be lagging behind the latest release due to the review process.*

Cheers! 🎉

## New Contributors
* @nicodier made their first contribution in https://github.com/immich-app/immich/pull/519

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.25.0_35-dev...v1.26.0_36-dev

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.26.0_36-dev'>v1.26.0_36-dev</a>.</em>

**Comment by cliffr39** [comment]
Once I backup from phone to Immich, how do I download them all from Immich without going to each individual image (to local storage on desktop to burn onto blu-ray)?

**Reply by alextran1502** [reply]
The location of the files are stored where you put in value of `UPLOAD_LOCATION` in the .env file

**Comment by cliffr39** [comment]
I saw that just was hoping for an option from the web interface like google
photos has

**Reply by alextran1502** [reply]
Like the export option?

**Comment by cliffr39** [comment]
Similar yes. Where I could select several random images or several full
days and click download instead of having to go into each one to
download them. Makes it easier than scp especially since you can see them
better to select which you want. My goal was to create a full copy of the
album on local computer storage to burn onto blank blu-ray archives
